### PR TITLE
added tracking to search autocomplete

### PR DESCRIPTION
**Pull request by bentedder**
adding in tracking for autocomplete search results.

### CODE REVIEW

File: app/assets/javascripts/lib/core/nav_search.js [hunk 1]
```diff
@@ -53,7 +53,13 @@ define([ "jquery", "autocomplete" ], function($, Autocomplete) {
   }
 
   NavSearch.prototype.onItem = function(el) {
-    window.location = $(el).attr("href");
+    var $el = $(el);
+    window.location = $el.attr("href");
+    window.lp.analytics.api.trackEvent({
```
Comment (jcreamer898) [inline app/assets/javascripts/lib/core/nav_search.js]: Not sure, but you MIGHT want to do the `window.location` part after the track event. Could run into a race condition. In theory, the page will change the moment you set a location.

Comment (bentedder) [inline app/assets/javascripts/lib/core/nav_search.js]: good catch. will change.

File: app/assets/javascripts/lib/core/nav_search.js [hunk 2]
```diff
@@ -53,7 +53,13 @@ define([ "jquery", "autocomplete" ], function($, Autocomplete) {
   }
 
   NavSearch.prototype.onItem = function(el) {
-    window.location = $(el).attr("href");
+    var $el = $(el);
```
Comment (jcreamer898) [inline app/assets/javascripts/lib/core/nav_search.js]: Another, teeny tiny thing, but you could also store `$el.attr("href");` as a variable, so you're not calling `attr` twice.

```js
var $el = $(el),
      href = $el.attr("href");
```

Comment (remybach) [inline app/assets/javascripts/lib/core/nav_search.js]: Sorry to butt in, but I think just using `el.href` would be the quickest here.

**Comment by jcreamer898** [comment]
:ship:

**Pull request merged**